src/OneExplorer.ts

```diff
@@ -77,15 +71,15 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
   }
 
   getChildren(element?: OneNode): OneNode[]|Thenable<OneNode[]> {
-    if (this.oneTree === undefined) {
-      vscode.window.showInformationMessage('No ONE model or config in empty workspace');
+    if (!this.workspaceRoot) {
+      vscode.window.showInformationMessage('Cannot find workspace root');
       return Promise.resolve([]);
```

Comment:
(not a part of this PR) but would `Promise.reject()` be correct?

Reply:
![image](https://user-images.githubusercontent.com/17171963/165895184-f2160a7f-ee4d-4034-86be-fd06dd69e412.png)

Reply:
Please refer to this https://github.com/microsoft/vscode-extension-samples/blob/main/tree-view-sample/src/nodeDependencies.ts#L22-L25 codes.

I think we don't need to show error message. Instead we can show the information noti for this one.

Reply:
Yes, I took the same reference. Both cases looks fine. You can choose. Or maybe we can discuss another time.